texas-hearing-institute/components/TabNavigator.tsx

```diff
@@ -4,13 +4,23 @@ import { NavigationContainer } from '@react-navigation/native';
 import { Text } from 'react-native';
 import PracticeNavigator from '../pages/PracticeNavigator';
 import { SoundInventory } from '../pages/SoundInventory/SoundInventory';
+import OnboardingNavigator from '../pages/Onboarding/OnboardingNavigator';
 
-const Tab = createBottomTabNavigator();
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
```

Comment:
this must be a type, not an interface, as described in the react navigation typescript docs